File: 1786-count-the-number-of-consistent-strings/1786-count-the-number-of-consistent-strings.js
/**
 * @param {string} allowed
 * @param {string[]} words
 * @return {number}
 */
var countConsistentStrings = function(allowed, words) {
    let ans=0;

    words.forEach((cur) =>{
        for(letter of cur){
            if(!allowed.includes(letter)){
            ans++;
            break;
           }
        }
    });
    return words.length - ans;
};

File: 1786-count-the-number-of-consistent-strings/1786-count-the-number-of-consistent-strings.cpp
class Solution {
public:
    int countConsistentStrings(string allowed, vector<string>& words) {
        int arr[26]={0}, ans=0;
        for(char c:allowed)
            arr[c-'a']++;

        for(string word: words){
            for(char c: word){
                if(arr[c-'a'] == 0){
                    ans++;
                    break;
                }
            }
        }

        return words.size() - ans;
    }
};